quic/s2n-quic-transport/src/transmission/application.rs

```diff
@@ -130,6 +133,12 @@ impl<'a, S: Stream, Config: endpoint::Config> Normal<'a, S, Config> {
 
             self.path_manager.on_transmit(context);
 
+            // By default, unreliable datagrams are sent before stream data. This can be
+            // configured by implementing a custom datagram sender and choosing to cede
+            // packet space for stream data.
+            self.datagram_manager
+                .on_transmit(context, self.stream_manager);
```

Comment:
nice

Reply:
Yeah good point. The logic to alternate between datagrams and streams hasn't been written yet, but that's what it will end up being. I'll change the comment to reflect that.